python/tidy/servo_tidy/tidy.py

```diff
@@ -382,6 +382,16 @@ def check_toml(file_name, lines):
         yield (0, ".toml file should contain a valid license.")
 
 
+def check_html_title_tag(file_name, lines):
+    if not file_name.endswith(".html"):
+        raise StopIteration
+    if type(lines) == str:
```

Comment:
I don't think we'll be needing this check. Since you've listed this under `line_checking_functions`, `lines` will always be a list of lines.